met-api/migrations/versions/0e043f976e2e_add_notes_to_submission.py

```diff
@@ -0,0 +1,38 @@
+"""add_notes_to_submission
+
+Revision ID: 0e043f976e2e
+Revises: 17bae2d586fb
+Create Date: 2023-01-10 12:18:43.395103
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from datetime import datetime
+
+
+# revision identifiers, used by Alembic.
+revision = '0e043f976e2e'
+down_revision = '17bae2d586fb'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('submission', sa.Column('notify_email', sa.Boolean(), nullable=True))
+    comment_status_table = sa.table('comment_status',
+        sa.Column('id', sa.Integer()),
+        sa.Column('status_name', sa.String(length=50)),
+        sa.Column('description', sa.String(length=50)),
+        sa.Column('created_date', sa.DateTime()),
+        sa.Column('updated_date', sa.DateTime()))
+    op.bulk_insert(comment_status_table, [
+        {'id': 4, 'status_name': 'Needs further review', 'description': 'Comment needs further review', 'created_date': datetime.utcnow(), 'updated_date': datetime.utcnow()},
```

Comment:
I mentioned this in the other PR i don't think we need a new status. we can chat about this